### PR TITLE
Add clip studio paint

### PR DESCRIPTION
**Pull request by jutonium**
Please review

### CODE REVIEW

**Comment by choules** [comment]
I was able to run the AutoPKG workflow to import the German version. The resulting package was successfully installed and uninstalled using Munki.

**Comment by a-vogel** [comment]
Clip Studio Paint also installs Clip Studio. This is a management application like Creative Cloud. Since the only other available software is Clip Studio Modeler that requires Clip Studio paint, it seems okay to bundle Clip Studio with Clip Studio Paint. Clip Studio Modeler (if needed) should be an update_for for Clip Studio Paint and is not covered by this recipe.

File: ClipStudioPaint/ClipStudioPaint.pkg.recipe [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+<dict>
+	<key>Description</key>
+	<string>Repackages Clip Studio Paint to remove the language selection step from the installation. The language can be chosen by setting the LANGUAGE variable to german, english, spanish, or french, japanese, korean, or chinese_tc.</string>
+	<key>Identifier</key>
+	<string>com.github.wycomco.pkg.ClipStudioPaint</string>
+	<key>Input</key>
+	<dict>
+		<key>LANGUAGE</key>
+		<string>english</string>
+		<key>NAME</key>
```
[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.pkg.recipe:13]: Yes, line 231

[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.pkg.recipe:13]: It is used in two of the FileCreator Processors for setting the language of the installation.

File: ClipStudioPaint/ClipStudioPaint.munki.recipe [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+			<key>description</key>
+			<string>Clip Studio Paint is a versatile digital painting program that is ideal for rendering and inking with its many useful and unique features.</string>
+			<key>developer</key>
+			<string>Celsys</string>
+			<key>display_name</key>
+			<string>Clip Studio Paint</string>
+			<key>minimum_os_version</key>
+			<string>10.13.0</string>
+			<key>name</key>
+			<string>%NAME%_%LANGUAGE%</string>
```
[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.munki.recipe:36]: Accepted
Changed

File: ClipStudioPaint/ClipStudioPaint.munki.recipe [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Comment</key>
+	<string>Even though the current version ist named 1.5 it has a ShortVersionString of 1.11.8.</string>
```
[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.munki.recipe:6]: Updated the comment.

File: ClipStudioPaint/ClipStudioPaint.munki.recipe [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+	<string>com.github.wycomco.munki.ClipStudioPaint</string>
+	<key>Input</key>
+	<dict>
+		<key>LANGUAGE</key>
+		<string>english</string>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/%NAME%</string>
+		<key>NAME</key>
+		<string>ClipStudioPaint</string>
+		<key>pkginfo</key>
```
[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.munki.recipe:19]: Accepted.
Done.

[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.munki.recipe:19]: Accepted
Done

File: ClipStudioPaint/ClipStudioPaint.pkg.recipe (hunk 1 above)
[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.pkg.recipe:13]: Required for english as well as any other language.

**Comment by jutonium** [comment]
Issues resolved in [PR8](https://github.com/autopkg/wycomco-recipes/pull/8)

File: ClipStudioPaint/ClipStudioPaint.pkg.recipe (hunk 1 above)
[the remark this answers is not in the document]
Comment (choules) [reply ClipStudioPaint/ClipStudioPaint.pkg.recipe:13]: I meant the NAME var, not the language.

[the remark this answers is not in the document]
Comment (jutonium) [reply ClipStudioPaint/ClipStudioPaint.pkg.recipe:13]: It's used for the name of the generated package.

**Pull request merged**